### PR TITLE
Solidity Summit 2023

### PR DESCRIPTION
**Pull request by NunoFilipeSantos**

### CODE REVIEW

File: src/events/solidity-summit-2023.md [hunk 1]
```diff
@@ -68,9 +68,12 @@ This year, presentations and discussions will evolve around the following topics
 - Layer 2 support, cross-chain support
 - Mechanisms / programming patterns (on the EVM) like mutexes, state machines, upgradeable contracts, libraries, create2, etc.
 
-## Registrations will open soon.
+## Registrations are open!
 
-Tickets for the Solidity Summit 2023 are free and given out on an application basis. Applications will open soon.
+Tickets for the Solidity Summit 2023 are free and given out on an application basis.
+Applications are open.
```
Comment (wackerow) [inline src/events/solidity-summit-2023.md:74]: This line break won't render, so you just put in all on one line

```suggestion
Tickets for the Solidity Summit 2023 are free and given out on an application basis. Applications are open.
```

Or if you want a line break (new paragraph), we can add another line:

```suggestion
Tickets for the Solidity Summit 2023 are free and given out on an application basis.

Applications are open.
```

**Review by wackerow — APPROVED** [review]
Looks good! Just noted one small clean-up suggestion.

I put up PR #38 which will rename this file, but the git merge should handle that without an issue.

**Comment by wackerow** [comment]
@NunoFilipeSantos Got the `main` branch build working... just updated this PR/branch with the latest, should be good to go!

**Comment by franzihei** [comment]
Added a new header picture and made small edits to the copy.

**Pull request merged**